_Classes/DeusEx/Classes/DeusExWeapon.uc

```diff
@@ -51,7 +51,7 @@ enum ELockMode
 };
 
 var bool				bReadyToFire;			// true if our bullets are loaded, etc.
-var() int				LowAmmoWaterMark;		// critical low ammo count
+var() travel int				LowAmmoWaterMark;		// critical low ammo count
```

Comment:
wait, isn't this basically supposed to be a const?

I see no reason why it should ever change during gameplay